src/WebUI/WorkLoads/WorkloadDetails.scss

```diff
@@ -1,3 +1,7 @@
 .workload-details-card .workload-full-details-table {
     padding-top: 10px;
+}
+
+.runs-bullet {
+    margin: 0 6px;
 }
```

Comment:
is this too generic, or are we okay?

Reply:
should be fine. I don't think anyone will override this accidentally, as it is a span